src/schema/basic_classes.yaml

```diff
@@ -414,6 +414,11 @@ classes:
     is_a: PlannedProcess
     aliases:
       - analysis
+    structured_aliases:
+      workflow_execution_class:
+       contexts:
+         -  https://bitbucket.org/berkeleylab/jgi-jat
+       predicate: EXACT_SYNONYM
```

Comment:
I had imagined putting this structure_alias on the `type` slot in this class' `slot_usage`